packages/jest-coverage/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@jest/coverage",
```

Comment:
this module does not (and should not) have to live here. Maybe we can make it part of `istanbul-lib-instrument`? All it does is start and stop the coverage collection using the v8 inspector API.

/cc @bcoe @coreyfarrell

Reply:
I don't have time to really look at this currently but I wouldn't object to creating a new module under the istanbuljs umbrella to accomplish this.  Maybe `@istanbuljs/v8-coverage` would be a good name?  I think it should be a separate module from `istanbul-lib-instrument` which pulls in a bunch of babel modules.

FWIW under the istanbuljs org I would not want TS source code but I don't have a problem with having the module include an `index.d.ts` with public definitions.

Reply:
:+1: I honestly have less concerns about TypeScript, since I've been using it at work these days; would be happy to put up my hand and help take on maintenance burden for this one.

Reply:
Feel free to take the code from this. Or create the repo and I can push it there.

If you want a separate `d.ts.` file, here's the built definition:

```ts
/// <reference types="node" />
import { Profiler } from 'inspector';
export declare type V8Coverage = ReadonlyArray<Profiler.ScriptCoverage>;
export default class CoverageInstrumenter {
    private readonly session;
    startInstrumenting(): Promise<void>;
    stopInstrumenting(): Promise<V8Coverage>;
}
```

Reply:
For now I stuck it in https://github.com/SimenB/collect-v8-coverage